ArduPlane/quadplane.cpp

```diff
@@ -3600,7 +3620,7 @@ bool QuadPlane::in_vtol_land_final(void) const
  */
 bool QuadPlane::in_vtol_land_sequence(void) const
 {
-    return in_vtol_land_approach() || in_vtol_land_descent() || in_vtol_land_final();
+    return plane.control_mode == &plane.mode_qrtl || in_vtol_land_approach() || in_vtol_land_descent() || in_vtol_land_final();
```

Comment:
Looks quite scary, but I don't think this is a change in behavior, just a small efficiency gain?

Reply:
yes, and one less special case